.gitignore

```diff
@@ -69,46 +87,40 @@ PCbuild/arm64/
 PCbuild/obj/
 PCbuild/win32/
 Tools/unicode/data/
-.purify
-__pycache__
-autom4te.cache
-build/
-buildno
-config.cache
-config.log
-config.status
-config.status.lineno
-core
-!Tools/msi/core/
-db_home
-.hg/
-.idea/
-ipch/
-libpython*.a
-libpython*.so*
-libpython*.dylib
-libpython*.dll
-platform
-pybuilddir.txt
+/autom4te.cache
+/build/
+/buildno
+/config.cache
+/config.log
+/config.status
+/config.status.lineno
```

Comment:
Maybe combine these into `/config.status*`?

Reply:
I think `config.status.lineno` is the only one like this -- it looks like it comes from [these lines of `configure`](https://github.com/python/cpython/blob/3f4db4a0bab073b768fae958e93288bd5d24eadd/configure#L468)... where it checks if the current shell has the special variable `$LINENO`, and if not, then it runs a sed script on *itself* (?), turning all the `$LINENO` references into the actual line number, and sources the resulting script instead.

Or something like that; this is some fairly dense shell-scripting. Not sure the exact pathway that causes that to end up making a `config.status.lineno` rather than `configure.lineno`. In any case, I'm not seeing signs of a more general pattern of doing the same with other suffixes.

Given that just `config.status` is the main thing, and `config.status.lineno` is a fairly obscure portability hack and the only one of its kind, I think it's probably more confusing than it's worth to write a general pattern `/config.status*` for them.